migrate-mongo-config-example.ts

```diff
@@ -20,6 +20,7 @@ const config = {
 
     // The mongodb collection where the applied changes are stored. Only edit this when really necessary.
     changelogCollectionName: "changelog",
+    novu_api_key: "NOVU_API_KEY",
```

Comment:
question: do we really need this?

Reply:
We need it to be able to make the migration since we need the API key to have access to Novus's API